src/TreeDisplay/TreeTag.py

```diff
@@ -376,18 +376,7 @@ def tpRenderTABLE(self, id, root_url, url, state, substate, diff, data,
                     exp = i + 1
                     break
 
-            ####################################
-            # Mostly inline encode_seq for speed
-            s = compress(json.dumps(diff))
-            if len(s) > 57:
-                s = encode_str(s)
-            else:
-                s = b2a_base64(s)[:-1]
-                l = s.find('=')
-                if l >= 0:
-                    s = s[:l]
-            s = s.translate(tplus)
```

Comment:
`encode_str` [already takes care of it](https://github.com/zopefoundation/DocumentTemplate/pull/12/files/914fcd8fbc2e1a0188922cb70727bb9ac488480a#diff-75123b3701bcec58a795ebe5b2baf2f6R635).  The old code should've only been doing that in the "inlined" branch (which was an insane micro-optimization, anyway).